core/src/main/java/io/questdb/griffin/SqlCompiler.java

```diff
@@ -1611,6 +1668,95 @@ private void validateTableModelAndCreateTypeCast(
             throw SqlException.position(0).put("timestamp is not defined");
         }
     }
+    
+    private CompiledQuery sqlBackup(SqlExecutionContext executionContext) throws SqlException {
+		final CharSequence tok = SqlUtil.fetchNext(lexer);
+		if (null != tok) {
+			if (Chars.equalsLowerCaseAscii(tok, "table")) {
```

Comment:
this can be replaced with existing `expectKeyword` call. It is a fairly standard thing to check in parser/compiler